src/sb_pointcloud_processing/rosconsole.config

```diff
@@ -0,0 +1,8 @@
+#
+#   rosconsole will find this file by default at $ROS_ROOT/config/rosconsole.config
+#
+#   You can define your own by e.g. copying this file and setting
+#   ROSCONSOLE_CONFIG_FILE (in your environment) to point to the new file
+#
+log4j.logger.ros=DEBUG
+log4j.logger.ros.roscpp.superdebug=WARN
```

Comment:
What's this for? If it's just to force specific debugging settings, we should remove it.